services/squareone/values-idfprod.yaml

```
@@ -12,9 +12,10 @@ squareone:
     - name: "pull-secret"
   config:
     siteName: "Rubin Science Platform"
-    # broadcastMarkdown: |
-    #   Write a markdown-formatted broadcast notification.
-
+        siteName: "Rubin Science Platform @ data-int"
+        # broadcastMarkdown: |
+        #    Patch Thursday **today** 3pm-5pm Pacific / 22:00-00:00 UT [(*what is RSP Patch Thursday?)*](https://ls.st/9vz)
+        
```

The linters are reporting an indentation inconsistency (our YAML standard is 2 spaces for an indent). It also looks like the `siteName` config from data-int got copied over? I’m also moving the italics for consistency:

```suggestion
  # broadcastMarkdown: |
  #    Patch Thursday **today** 3pm-5pm Pacific / 22:00-00:00 UT [*(what is RSP Patch Thursday?)*](https://ls.st/9vz)]

```